include/torch-mlir/Dialect/Torch/Utils/Utils.h

```diff
@@ -9,11 +9,11 @@
 #ifndef TORCHMLIR_DIALECT_TORCH_UTILS_H
 #define TORCHMLIR_DIALECT_TORCH_UTILS_H
 
+#include "mlir/IR/PatternMatch.h"
```

Comment:
Because we have a `rewriter` variable of class `PatternRewriter` https://github.com/llvm/torch-mlir/blob/41bcdcdbe910ebcd7b0521ae64effaf42c948988/include/torch-mlir/Dialect/Torch/Utils/Utils.h#L26, if we remove this header file, it throws an error.